ceph/rados/core_workflows.py

```diff
@@ -1487,3 +1487,17 @@ def list_orch_services(self, service_type=None):
         for service in orch_ls_op:
             service_name_ls.append(service["service_name"])
         return service_name_ls
+
+    def check_host_status(self, hostname) -> bool:
```

Comment:
if the host being rebooted is the node, where Cephadm shell resides, this would fail.

Do we want to configure this command to be run on Client? Client reboots are seldom done. So should be safe.

Let me know what you think.

Reply:
Thanks for pointing that out, I have changed it to client node. Please review, RHCS 5.3 logs are with the latest changes.